### ISSUE TITLE
Enable data flow between apps and locally stored data FOR volunteers

### ISSUE THREAD

**Issue by magdziarek** [biodiversitydata-se/biocollect#153]
Most of those who have asked would like to have some sort of function that makes it easy for them to also report their observations to Artportalen.

In general words. If they enter data in Biocollect to start with, then they would like to extract/export data to some other format (like excel) which is easy to import into Artportalen. The main thing is that they would like to enter data just once (into Biocollect), but then be able to use the entered data (now in Biocollect) also for reporting to Artportalen or for storing "at home in own systems" etc.

Then there are of course other variants of this as well. Entering data in excel/Artportalen/own systems to start with and then wanting to be able to import data into Biocollect from these "original sources".

The bottom line in all cases is that more and more people (not many so far, but an increasing number anyway) are asking for systems where they can enter data once and then use the entered data in many different sistuations. We are of course not there at the moment, but it might be good to have this in mind for the future.
